docs/guides/local-testing-mode.ipynb

```diff
@@ -37,12 +39,36 @@
     "\n",
     "- **Fake backends**: The [fake backends](/docs/api/qiskit-ibm-runtime/fake-provider)  in ``qiskit_ibm_runtime.fake_provider`` mimic the behaviors of IBM&reg; QPUs by using QPU snapshots. The QPU snapshots contain important information about the QPU, such as the coupling map, basis gates, and qubit properties, which are useful for testing the transpiler and performing noisy simulations of the QPU. The noise model from the snapshot is automatically applied during simulation.\n",
-    "- **Aer simulator**: Simulators from [Qiskit Aer](simulate-with-qiskit-aer) provide higher-performance simulation that can handle larger circuits and [custom noise models](/docs/guides/build-noise-models). It also supports Clifford simulation mode, which can efficiently simulate Clifford circuits with a large number of qubits.\n",
+    "- **Aer simulator**: Simulators from [Qiskit Aer](/docs/guides/simulate-with-qiskit-aer) provide higher-performance simulation that can handle larger circuits and [custom noise models](/docs/guides/build-noise-models). A list of simulation method options are available when you use `AerSimulator` in local testing mode. See the [Clifford simulation mode example](#clifford-sim), which demonstrates how to efficiently simulate Clifford circuits with a large number of qubits.\n",
+    "\n",
+    "    <details>\n",
+    "    <summary>**List of simulation methods available from Qiskit Aer**</summary>\n",
+    "\n",
+    "    See the [`AerSimulator`](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator) documentation for more information.\n",
+    "\n",
+    "    * ``\"automatic\"``: Default simulation method. Select the simulation method automatically, based on the circuit and noise model.\n",
+    "\n",
+    "    * ``\"statevector\"``: A dense statevector simulation that can sample measurement outcomes from *ideal* circuits with all measurements at end of the circuit. For noisy simulations, each shot samples a randomly sampled noisy circuit from the noise model.\n",
+    "\n",
+    "    * ``\"density_matrix\"``: A dense density matrix simulation that can sample measurement outcomes from *noisy* circuits with all measurements at end of the circuit.\n",
+    "\n",
+    "    * ``\"stabilizer\"``: An efficient Clifford stabilizer state simulator that can simulate noisy Clifford circuits if all errors in the noise model are also Clifford errors.\n",
+    "\n",
+    "    * ``\"extended_stabilizer\"``: An approximate simulator for Clifford + T circuits based on a state decomposition into ranked-stabilizer state. The number of terms grows with the number of non-Clifford (T) gates.\n",
+    "\n",
+    "    * ``\"matrix_product_state\"``: A tensor-network statevector simulator that uses a Matrix Product State (MPS) representation for the state. This can be done either with or without truncation of the MPS bond dimensions depending on the simulator options. The default behavior is no truncation.\n",
```

Comment:
```suggestion
    "    * ``\"matrix_product_state\"``: A tensor-network statevector simulator that uses a Matrix Product State (MPS) representation for the state. This can be done with or without truncating the MPS bond dimensions, depending on the simulator options. The default behavior is no truncation.\n",
```